asset-transfer-private-data/chaincode-go/META-INF/statedb/couchdb/collections/collectionMarbles/indexes/indexOwner.json

```diff
@@ -0,0 +1 @@
+{"index":{"fields":["docType","owner"]},"ddoc":"indexOwnerDoc", "name":"indexOwner","type":"json"}
```

Comment:
"docType" in above  index definition should be "type" since the asset prop got rename

Reply:
done.